src/api/app/models/notification.rb

```diff
@@ -1,7 +1,7 @@
 class Notification < ApplicationRecord
   belongs_to :subscriber, polymorphic: true
 
-  serialize :event_payload, Hash
+  serialize :event_payload
```

Comment:
Yeah I think its better to remove the existing notifications from the DB. I'm pretty sure `Hash` is necessary here for the rest of the code to work.